### ISSUE TITLE
Several errors

### ISSUE THREAD

**Issue by webbird** [piwik/piwik#7417]
I've just installed Piwik in my local XAMPP environment and I am getting a bunch of errors.

`piwik\core\SettingsPiwik.php(421): Notice - Undefined index: instance_id - Piwik 2.11.2`
`piwik\core\ProxyHttp.php(252): Warning - filemtime(): stat failed for <pathto>\piwik/tmp/assets/asset_manager_global_css.css.deflate - Piwik 2.11.2`
`piwik\core\Plugin\Report.php(836): Notice - Undefined index: items - Piwik 2.11.2`

All I did until now is install, login, call [Administration].

**Comment by webbird** [comment]
System check results:

![2015-03-12 15_11_24-piwik administration](https://cloud.githubusercontent.com/assets/147354/6619366/19de561a-c8ca-11e4-92f6-3cdf2f74abf2.png)

All other are green, too.

**Comment by mnapoli** [comment]
Which Piwik version have you installed? The first line number do not match the error, I believe you might not have the latest version. Or could you try installing the beta?

In any case these logs are weird because the warnings are voluntarily silenced, so they shouldn't appear in the logs.

Maybe it's because xdebug is installed and [`xdebug.scream`](http://xdebug.org/docs/all_settings) is set to 1, which forces all the error messages to appear… In that case those error messages are not actual errors and will not appear if Piwik is installed in a production server (i.e. not a development PHP installation).

In short, this is probably a PHP configuration issue (XAMPP is pre-configured in development mode) and not a problem with Piwik.
